doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,46 @@
+# User Guides for the generic Virtual Schema for Document Files
+
+You cannot directly use this adapter! Use one of the file source-specific dialects instead. 
+For a complete list of all official dialects, check the [README](../../README.md).
+If you need something different, you can also [implement a custom dialect](howto_create_new_dialect.md).
+
+This user guide only covers the generic parts of the file virtual schema dialects.
+If you want to get started or need data source-specific information, start with the dialect-specific user guide.
+
+# Document Types
+
+This Virtual Schema implements support for different document types (JSON, JSON-Lines, ...).
+Don't confuse that with the different data source specific dialects.
+Each dialect implements a different file provider (S3, BucketFS, ...). 
+With each of these dialects, you define mappings for the different data types listed here.
+
+You can not add different data types by adding a dialect.
```

Comment:
```suggestion
You cannot add different data types by adding a dialect.
```

Reply:
This sentence is also not clear to me. What does data type mean here?

Reply:
```suggestion
You can not add support for different document types (like JOSN) by adding a dialect.
```

Reply:
:+1: